g11n-ws/vip-manager-lite-i18n/src/main/resources/application-bundle.properties

```diff
@@ -28,6 +28,9 @@ server.trace.enable=false
 server.shutdown=graceful
 spring.lifecycle.timeout-per-shutdown-phase=30s
 
+#translation update switch
+translation.update.enable=true
```

Comment:
According to our discussion, this is false by default?

Reply:
change it to false